libsql-server/src/schema/scheduler.rs

```diff
@@ -773,6 +789,9 @@ async fn step_job_run_success(
                         txn.pragma_update(None, "schema_version", job_id)?;
                         // update schema version to job_id?
                         txn.commit()?;
```

Comment:
There are 3 statements here that can return with error and I believe we won't re-enable foreign_keys in such case, will we?